File: src/main/java/com/tva/mk/rsp/SingleRsp.java
package com.tva.mk.rsp;

import com.fasterxml.jackson.annotation.JsonProperty;

public class SingleRsp extends BaseRsp {
	// region -- Fields --

	@JsonProperty(value = "result")
	private Object result;

	// end

	// region -- Get set --

	public Object getResult() {
		return result;
	}

	public void setResult(Object result) {
		this.result = result;
	}

	// end

	// region -- Methods --

	public SingleRsp() {
	}

	public SingleRsp(String status, String message, Object result) {
		super(status, message);
		this.result = result;
	}

	// end
}

File: dotnet/MoneyKeeper.Main/MoneyKeeper.Main/Rsp/SingleRsp.cs
using Newtonsoft.Json;

namespace MoneyKeeper.Main.Rsp
{
    /// <summary>
    /// Single response
    /// </summary>
    public class SingleRsp : BaseRsp
    {
        #region -- Properties --

        [JsonProperty("result")]

        public object Result;

        #endregion
    }
}